spec/features/logging/view_logs_for_a_username_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'features/support/sign_up_helpers'
+
+describe "View logs for a username" do
+  let(:user) { create(:user, :confirmed, :with_organisation) }
+  let(:username) { "AAAAA" }
+  let(:logs) do
+    [
+      {
+        "ap" => "",
+        "building_identifier" => "",
+        "id" => 1,
+        "mac" => "",
+        "siteIP" => "",
+        "start" => "2018-10-01 18:18:09 +0000",
+        "stop" => nil,
+        "success" => true,
+        "username" => username
+      },
+      {
+        "ap" => "",
+        "building_identifier" => "",
+        "id" => 1,
+        "mac" => "",
+        "siteIP" => "",
+        "start" => "2018-10-01 18:18:09 +0000",
+        "stop" => nil,
+        "success" => true,
+        "username" => username
+      }
+    ]
+  end
+
+  before do
+    stub_request(:get, "http://govwifi-logging-api.com/authentication/events/search?username=#{username}").
```

Comment:
This looks like a good approach to me, webmocking the API in the acceptance spec.

Is it not working? Or were you concerned this isn't the right approach?